llvm/docs/LangRef.rst

```diff
@@ -159,7 +159,7 @@ There are two kinds of escapes.
 * ``\\`` represents a single ``\`` character.
 
 * ``\`` followed by two hexadecimal characters (0-9, a-f, or A-F)
-  represents the byte with the given value (e.g. \x00 represents a
+  represents the byte with the given value (e.g. ``\00`` represents a
```

Comment:
<img width="238" alt="Screenshot 2025-04-29 at 8 32 29 PM" src="https://github.com/user-attachments/assets/55f6ad30-4e32-41b4-a61b-8b148ee4af43" />

The current rendering is incorrect.